setup.py

```diff
@@ -54,9 +60,9 @@ def _get_dbt_core_version():
 dbt_core_version = _get_dbt_core_version()
 description = """The Apache Spark adapter plugin for dbt"""
 
-odbc_extras = ["pyodbc~=4.0.30"]
+odbc_extras = ["pyodbc~=4.0.39"]
 pyhive_extras = [
-    "PyHive[hive]>=0.6.0,<0.7.0",
+    "PyHive[hive_pure_sasl]~=0.7.0",
```

Comment:
This needs to line up with `requirements.txt`. Our CI test runs install `requirements.txt`, but our build would just use `setup.py`.